src/Apps/Artwork/Components/ArtworkVideoPlayer/ArtworkVideoPlayer.tsx

```diff
@@ -35,10 +32,11 @@ const ArtworkVideoPlayer: FC<ArtworkVideoPlayerProps> = ({
       {...rest}
     >
       <ResponsiveBox
-        mx={[0, "70px"]}
```

Comment:
I realize this is for the UI — I'm wondering if it's needed. Going to come back to this in the morning.

I think we could just show/hide the arrows like we do in the zoom view.

Reply:
Re-added it; I'm going to circle back to get rid of this though.